.ruby-version

```diff
@@ -0,0 +1 @@
+2.2.2
```

Comment:
@tmtrademarked Are there any fast-follows we can tag to find a way to update this? Ruby 2.2 has been unsupported since March https://www.ruby-lang.org/en/news/2018/06/20/support-of-ruby-2-2-has-ended/

😱 😞

Reply:
This project is kind of dead - the goal is to cut over to the `marketplace-client` instead. Unfortunately, that's a somewhat bigger project. (That project has a real pipeline, etc, which makes it easier to work with - and actually supports versioning, etc!)